app/src/main/res/values-nl/strings.xml

```diff
@@ -1,66 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">Eenvoudige Galerij</string>
+    <string name="app_name">Eenvoudige galerij</string>
     <string name="app_launcher_name">Galerij</string>
     <string name="edit">Bewerken</string>
     <string name="open_camera">Camera</string>
     <string name="hidden">(verborgen)</string>
     <string name="pin_folder">Map vastzetten</string>
     <string name="unpin_folder">Map losmaken</string>
-    <string name="pin_to_the_top">Pin to the top</string>
-    <string name="show_all">Platte weergave</string>
+    <string name="pin_to_the_top">Bovenaan vastzetten</string>
+    <string name="show_all">Alles weergeven</string>
     <string name="all_folders">Alle mappen</string>
     <string name="folder_view">Mapweergave</string>
     <string name="other_folder">Andere map</string>
-    <string name="show_on_map">Op kaart tonen</string>
-    <string name="unknown_location">Locatie onbekend</string>
-    <string name="no_map_application">Geen app voor kaarten gevonden</string>
-    <string name="no_camera_app_found">Geen app voor camera gevonden</string>
+    <string name="show_on_map">Weergeven op kaart</string>
+    <string name="unknown_location">Onbekende locatie</string>
+    <string name="no_map_application">Geen kaarten-app gevonden</string>
+    <string name="no_camera_app_found">Geen camera-app gevonden</string>
     <string name="increase_column_count">Meer kolommen</string>
     <string name="reduce_column_count">Minder kolommen</string>
-    <string name="change_cover_image">Afbeelding voor omslag veranderen</string>
+    <string name="change_cover_image">Omslagafbeelding wijzigen</string>
     <string name="select_photo">Foto selecteren</string>
     <string name="use_default">Standaard gebruiken</string>
     <string name="volume">Volume</string>
     <string name="brightness">Helderheid</string>
-    <string name="do_not_ask_again">Onthouden voor deze sessie</string>
-    <string name="lock_orientation">Schermrotatie vergrendelen</string>
-    <string name="unlock_orientation">Schermrotatie ontgrendelen</string>
+    <string name="do_not_ask_again">Niet opnieuw vragen (in deze sessie)</string>
```

Comment:
No

Reply:
That's the Dutch (Netherlands) way of translating this. Why don't you start a translation for Dutch (Belgium) if you want to use your style?